tsconfig.json

```diff
@@ -2,14 +2,14 @@
 	"compilerOptions": {
 		"baseUrl": "src",
 		"outDir": "./dist",
-		"module": "ESNext",
+		"module": "node16",
 		"target": "ES2017",
 		"jsx": "react-jsx",
 		"lib": ["ESNext", "ES5", "ES6", "DOM", "DOM.Iterable"],
 		"sourceMap": true,
 		"strict": true,
 		"allowJs": false,
-		"moduleResolution": "node",
+		"moduleResolution": "node16",
 		"forceConsistentCasingInFileNames": true,
```

Comment:
ts documentation highly recommends that libraries use node16 or nodenext. This does not mean that our library has to use v16 of node.